### DISCUSSION TITLE
My school doesn't appear on the map.

### DISCUSSION THREAD

**Discussion by atozuser0224** [community/community#109634] [GitHub Education]
### Hi everyone,

I am a student at Jinryang High School in South Korea. I tried to get a student pack using the account 23010503@jinryang.hs.kr, but I couldn't send the application because my school doesn't exist in Microsoft Maps.
help.

**Comment by namith1003** [comment]
![image](https://github.com/community/community/assets/65733592/a3e95693-90f7-4817-8c8f-d06b0ec27542)


to me, your high school does in fact exist in their choices I'm not fully sure why this may be maybe try it again or try from a different device or an incognito tab.

Hopefull this helps and answers your question, if there is anything else please don't mind contacting me,

A.A Namith Nimlaka
